src/util/FactorUtil.js

```diff
@@ -24,69 +24,69 @@ function (Okta, TimeUtil) {
   var factorData = {
     'OKTA_VERIFY': {
       label: 'factor.totpSoft.oktaVerify',
-      description: Okta.loc('factor.totpSoft.description', 'login'),
+      description: _.partial(Okta.loc, 'factor.totpSoft.description', 'login'),
       iconClassName: 'mfa-okta-verify',
       sortOrder: 1
     },
     'OKTA_VERIFY_PUSH': {
       label: 'factor.oktaVerifyPush',
-      description: Okta.loc('factor.push.description', 'login'),
+      description: _.partial(Okta.loc, 'factor.push.description', 'login'),
       iconClassName: 'mfa-okta-verify',
       sortOrder: 1
     },
     'U2F': {
       label: 'factor.u2f',
       description: (brandName) => {
         return brandName ?
-          Okta.loc('factor.u2f.description.specific', 'login', [brandName]) :
```

Comment:
is this change necessary since `description` is already a function?

Reply:
Yeah, I think for cases with brandName we don't need the _.partial since its a function.